server/models/Profile.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require("mongoose")
+
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true
+  },
+  lastName: {
+    type: String,
+    required: true
+  },
+  // To allow similar names
+  id: {
+    type: Number,
+    required: true
+  },
+  availability: {
+    unavailableDates: [Date],
+    extraAvailability: [Date],
+    weeklyAvailability: {
+      // These will be strings of date time ranges
+      monday: [String],
+      tuesday: [String],
+      wednesday: [String],
+      thursday: [String],
+      friday: [String],
+      saturday: [String],
+      sunday: [String]
+    }
+  },
+  description: String,
+  sitter: {
+    isSitting: { type: boolean, required: true },
+    hourlyRate: Number,
+    Rating: Number
+  },
+  requests: [{
```

Comment:
Similar to the case above, we are going to have a separate model for requests as well, you can safely remove this field too!